packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -17,6 +19,8 @@ import {
   // NotFound
 } from './errors';
 
+const HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
Does the `https-proxy-agent` module support browsers? It does not mention it in the top level documentation.

Should stick with `import` in the sdk instead of `require(..)`.